apps/glusterfs/dbattribute_entry.go

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright (c) 2017 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
```

Comment:
This seems to be a very general, very thin wrapper around the db functions. What's the point?

Reply:
As this is a new bucket, all the functions that operate on a bucket need to be implemented again.

Reply:
Got it. This looks so boilerplate key/value access that I was briefly confused... ;-)